### ISSUE TITLE
Formatting working perfectly but removes the theme colors.

### ISSUE THREAD

**Issue by iyashpal** [shufo/vscode-blade-formatter#56]
## VS Code
Version: 1.52.1 (user setup)
Commit: ea3859d4ba2f3e577a159bc91e3074c5d85c0523
Date: 2020-12-16T16:34:46.910Z
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042

## Template

Happening in all blade files. 

## Context (Environment)

When extension is enabled:
![image](https://user-images.githubusercontent.com/19922707/106103361-d56a3d00-6166-11eb-8668-997a88efdf70.png)

When extension is disabled:
![image](https://user-images.githubusercontent.com/19922707/106103642-4578c300-6167-11eb-9b38-0b4ec7f5113c.png)

- write your environment

This should not remove the colors from editor view. I thought the issue was from theme but after switching to several theme the issue remains the same. 

Any other informations:

Current Theme I am using: [Tokyo Night](https://github.com/enkia/tokyo-night-vscode-theme)

**Comment by iyashpal** [comment]
@shufo when I can get an update on this? I am eagerly waiting for this update so I can use it on regular basis. Please Please fix this.

**Comment by shufo** [comment]
@iyashpal 
Sorry for late reply.

It seems something affects your theme but vscode-blade-formatter do nothing for vscode color setting so tbh I can't figure out what was the issue.

I also tried Tokyo Night + vscode-blade-formatter but color theme seems correctly worked.

![image](https://user-images.githubusercontent.com/1641039/112177006-ef427080-8c3b-11eb-9596-0ea557a7247c.png)

Can you give any more information? 

e.g. extension list
```bash
$ code --list-extensions
```

**Comment by iyashpal** [comment]
![image](https://user-images.githubusercontent.com/19922707/112182776-99fa6500-8c23-11eb-8ff9-9be8e151d653.png)

**Comment by iyashpal** [comment]
I am using the windows OS

**Comment by iyashpal** [comment]
@shufo I have disabled all the extensions and enabled only Blade Formatter but the error is same. Please check
![image](https://user-images.githubusercontent.com/19922707/112743892-e8f42180-8fb8-11eb-967d-84551b086b88.png)

**Comment by shufo** [comment]
@iyashpal 
ah, I  might recognized. It seems your extension list have no syntax highlighter for blade. Since vscode-blade-formatter has no syntax highlighting feature so has to install highlighter separately like [laravel-blade](https://marketplace.visualstudio.com/items?itemName=cjhowe7.laravel-blade) or [Laravel Blade Snippets](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade).

**Comment by iyashpal** [comment]
Is it possible to have highlighting feature to the extension? Because I don't want to use extensions that are not useful for me.

Thank you

**Comment by shufo** [comment]
Currently there is no plan to support highlight syntax because this project scope is limited to formatting template and existing extension like `laravel-blade` works properly. But if there is need to support highlight syntax with this extension, I'll take consider it!

**Comment by MACscr** [comment]
Right, the problem isnt that we want the plugin to provide syntax highlighting, but that we dont want it to remove our themes built in highlighting features.

**Comment by sanskar-mk2** [comment]
Is it possible that the extension does not remove highlighting and only work as a "formatter". Thank you. I am aware that there is workaround for it as mentioned above, but it seemed hacky. It would be better user experience if our color/theme/linting stays intact when we install this extension.

**Comment by shufo** [comment]
@sanskar-mk2 
This Extension doesn't implement any Syntax Highlighting, so I don't know the cause of problem tbh, but it seems to have issue when installed with specific Extensions.

can you provide list of Extensions you have problem by running following command?

```bash
code --list-extensions
```

**Comment by whmountains** [comment]
The issue in my case was that blade was I Was missing the laravel blade snippets plugin.  It would be awesome if the Readme could mention that this was required.
